queries.sql

```diff
@@ -40,4 +40,12 @@ UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg<0;
  SELECT AVG(weight_kg) FROM animals;
  SELECT neutered, MAX(escape_attempts) AS max_escape_attempts FROM animals GROUP BY neutered;
  SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
- SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
\ No newline at end of file
+ SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
+
+ SELECT animals.name,owners.full_name FROM animals INNER JOIN owners ON animals.owner_id=owners.id WHERE owners.full_name='Melody Pond';
+ SELECT animals.name,species.name FROM animals JOIN species ON animals.species_id=species.id WHERE species.name='Pokemon';
+ SELECT owners.full_name,animals.name FROM animals RIGHT JOIN owners ON animals.owner_id=owners.id;
```

Comment:
- [ ] The screenshot of this queries is missing from the PR description, kindly add it as required. 👍

Reply:
### Here is the screenshot of the queries
![ownersjoin](https://github.com/DianaBeki/vet_clinic/assets/99320947/2b90eb99-17f9-459c-86d5-cca051bf4724)